docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1596,6 +1596,10 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
 
 The Redis server address that should be connected to.
 
+{{< admonition type="note" >}}
+For more information on Redis, refer to [Enable alerting high availability using Redis](https://grafana.com/docs/grafana/latest/alerting/set-up/configure-high-availability/#enable-alerting-high-availability-using-redis).
```

Comment:
Without this, "next" will link to "latest" instead of "next".

https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution

```suggestion
For more information on Redis, refer to [Enable alerting high availability using Redis](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-high-availability/#enable-alerting-high-availability-using-redis).
```